src/Goteo/Core/Helpers.php

```diff
@@ -77,17 +77,10 @@ function trace ($resource = null) {
         echo '<pre>' . print_r($resource, true) . '</pre>';
     }
 
-    /**
-     * Vuelca información sobre el recurso especificado de forma detallada.
-     */
-    function dump ($resource = null) {
-        echo '<pre>' . var_dump($resource) . '</pre>';
-    }
```

Comment:
Now there's a dump method and this one is aiming to replace the other one. Since it doesn't seem it was being used, I've simply removed it.